chia/util/file_keyring.py

```diff
@@ -252,7 +253,7 @@ def get_password(self, service: str, user: str) -> Optional[str]:
             return self._inner_get_password(service, user)
 
     @loads_keyring
-    def _inner_set_password(self, service: str, user: str, passphrase: str, *args, **kwargs):
+    def _inner_set_password(self, service: str, user: str, passphrase: str) -> None:
```

Comment:
Just making a note here since this is more than just hinting.  You reviewed all references to this?

Reply:
> You reviewed all references to this?

Jup